app/policies/transaction_policy.rb

```diff
@@ -5,22 +5,22 @@ def index?
 
   def show?
     user.administrator? ||
-      activity? && user.fund_manager?
+      activity?
```

Comment:
The `activity?` check can be removed here, it's not needed as everyone can do everything now (unless you want to keep it for if/when we re-introduce restrictions on what users can do?)

Reply:
Yes it could be and yes I would love to keep it and let the next PR remove it. Actually it sets it to a simpler `true` in all cases and defers to the activity policy: https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/pull/199/files#diff-fb8862afbee289b19cc09ac511fc0da4R3

Reply:
I'm just looking at the next PR so feel free to leave these in!